pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ArrayFunctions.java

```diff
@@ -74,6 +76,50 @@ public static int arrayIndexOfString(String[] values, String valueToFind) {
     return ArrayUtils.indexOf(values, valueToFind);
   }
 
+  @ScalarFunction
+  public static int[] arrayIndexOfAllInt(int[] value, int valueToFind) {
```

Comment:
do we need to support more type like long/float/double etc?

Reply:
Will do.